### ISSUE TITLE
Missing BTC icon creating invisible bitcoin observables during investigations. 

### ISSUE THREAD

**Issue by threathive** [yeti-platform/yeti#370]
### Description
Missing icon for Bitcoin address in the current flaticon pack is preventing a new visual node getting added to the investigation graph when adding a new BTC address. 

We simply need to update to the following files|lines in the "/core/web/frontend/staticfiles/yeti/js/graph.js" and "/core/web/frontend/staticfiles/yeti/css/flaticon.css"  as well as update the current icon set to have some type of BTC address logo. I would do a merge , but I don't have the tools on this current machine or even honestly know where to begin for adding an icon to the flaticon.* set. If I have time this week I will try, but maybe someone else knows how to do this really quick :)
 

```
// Define default icons
var icons = {
  'Observable.Ip': flaticon('\ue005'),
  'Observable.AutonomousSystem': flaticon('\ue00b'),
  'Observable.MacAddress': flaticon('\ue005'),
  'Observable.Certificate': flaticon('\ue024'),
  'Observable.CertificateSubject': flaticon('\ue024'),
  'Observable.Hostname': flaticon('\ue01E'),
  'Observable.Url': flaticon('\ue013'),
  'Observable.Email': flaticon('\ue01A'),
  'Observable.Text': flaticon('\ue022'),
  'Observable.File': flaticon('\ue021'),
  'Observable.Hash': flaticon('\ue00e'),
  'Observable.Path': flaticon('\ue00a'),
  'Entity.Malware': flaticon('\ue001'),
  'Entity.TTP': flaticon('\ue019'),
  'Entity.Company': flaticon('\ue002'),
  'Entity.Actor': flaticon('\ue017'),
  'Entity.Campaign': flaticon('\ue003'),
  'Indicator.Regex': flaticon('\ue015'),
};

var cssicons = {
  'Observable.Ip': 'flaticon-computer189',
  'Observable.AutonomousSystem': 'flaticon-earth213',
  'Observable.MacAddress': 'flaticon-computer189',
  'Observable.Certificate': 'flaticon-website17',
  'Observable.CertificateSubject': 'flaticon-website17',
  'Observable.Hostname': 'flaticon-server20',
  'Observable.Url': 'flaticon-links11',
  'Observable.Email': 'flaticon-message99',
  'Observable.Text': 'flaticon-typography2',
  'Observable.File': 'flaticon-text70',
  'Observable.Hash': 'flaticon-finger14',
  'Observable.Path': 'flaticon-document238',
  'Entity.Malware': 'flaticon-bug24',
  'Entity.TTP': 'flaticon-maths5',
  'Entity.Company': 'flaticon-building259',
  'Entity.Campaign': 'flaticon-businessman116',
  'Entity.Actor': 'flaticon-malware',
  'Indicator.Regex': 'flaticon-magnifying-glass40',
};

```
### Environment

Current master branch

### Steps to Reproduce

Check out the master branch and attempt to add a BTC address in an investigation.

#### Expected behavior

You have a visual node representing a BTC address.

#### Actual behavior

You will see when you select all nodes within the graph you have a BTC address. However there is visually no node. So it makes doing any type of investigation a complete mess.

**Comment by threathive** [comment]
Potential fix in https://github.com/yeti-platform/yeti/pull/377

**Comment by tomchop** [comment]
FTR, this was fixed in #378.
